src/snowflake/snowpark/types.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, length: Optional[int] = None) -> None:
         self.length = length
 
     def __repr__(self) -> str:
-        if self.length:
+        if self.length and self.length < self._MAX_LENGTH:
```

Comment:
@sfc-gh-yixie is this going to be a BCR?
https://github.com/snowflakedb/snowpark-python/pull/1129/files

Reply:
reverted